src/components/media_control/media_control.js

```diff
@@ -110,26 +125,36 @@ export default class MediaControl extends UIObject {
-      this.listenTo(this.container, Events.CONTAINER_PROGRESS, this.updateProgressBar)
-      this.listenTo(this.container, Events.CONTAINER_SETTINGSUPDATE, this.settingsUpdate)
-      this.listenTo(this.container, Events.CONTAINER_PLAYBACKDVRSTATECHANGED, this.settingsUpdate)
-      this.listenTo(this.container, Events.CONTAINER_HIGHDEFINITIONUPDATE, this.highDefinitionUpdate)
-      this.listenTo(this.container, Events.CONTAINER_LOADEDTEXTTRACK, this.ccAvailable.bind(this, true))
-      this.listenTo(this.container, Events.CONTAINER_MEDIACONTROL_DISABLE, this.disable)
-      this.listenTo(this.container, Events.CONTAINER_MEDIACONTROL_ENABLE, this.enable)
-      this.listenTo(this.container, Events.CONTAINER_ENDED, this.ended)
-      this.listenTo(this.container, Events.CONTAINER_VOLUME, this.onVolumeChanged)
-      if (this.container.playback.el.nodeName.toLowerCase() === 'video') {
-        // wait until the metadata has loaded and then check if fullscreen on video tag is supported
-        this.listenToOnce(this.container, Events.CONTAINER_LOADEDMETADATA, this.onLoadedMetadataOnVideoTag)
-      }
+    if (!this.container) {
+      return;
+    }
+
+    Mediator.on(`${this.options.playerId}:${Events.PLAYER_RESIZE}`, this.playerResize, this)
+
+    this.listenTo(this.container, Events.CONTAINER_PLAY, this.changeTogglePlay)
+    this.listenTo(this.container, Events.CONTAINER_PAUSE, this.changeTogglePlay)
+    this.listenTo(this.container, Events.CONTAINER_STOP, this.changeTogglePlay)
+    this.listenTo(this.container, Events.CONTAINER_DBLCLICK, this.toggleFullscreen)
+    this.listenTo(this.container, Events.CONTAINER_TIMEUPDATE, this.onTimeUpdate)
+    this.listenTo(this.container, Events.CONTAINER_PROGRESS, this.updateProgressBar)
+    this.listenTo(this.container, Events.CONTAINER_SETTINGSUPDATE, this.settingsUpdate)
+    this.listenTo(this.container, Events.CONTAINER_PLAYBACKDVRSTATECHANGED, this.settingsUpdate)
+    this.listenTo(this.container, Events.CONTAINER_HIGHDEFINITIONUPDATE, this.highDefinitionUpdate)
+    this.listenTo(this.container, Events.CONTAINER_LOADEDTEXTTRACK, this.ccAvailable.bind(this, true))
+    this.listenTo(this.container, Events.CONTAINER_MEDIACONTROL_DISABLE, this.disable)
+    this.listenTo(this.container, Events.CONTAINER_MEDIACONTROL_ENABLE, this.enable)
+    this.listenTo(this.container, Events.CONTAINER_ENDED, this.ended)
+    this.listenTo(this.container, Events.CONTAINER_VOLUME, this.onVolumeChanged)
+
+    var video = this.container.playback.el;
+    if (video.nodeName.toLowerCase() === 'video') {
+      // wait until the metadata has loaded and then check if fullscreen on video tag is supported
+      this.listenToOnce(this.container, Events.CONTAINER_LOADEDMETADATA, this.onLoadedMetadataOnVideoTag)
+
+      // watch for user to come out of fullscreen and remove the native controls that iOS tends to add
+      video.addEventListener('webkitendfullscreen', function() {
```

Comment:
@bikegriffith There's a problem with this implementation. Perhaps it should be implemented on the HTML5Video playback itself. There's an option for the playback that tells it to use the default video tag controls (`playback.controls`), and this code would not follow the configuration correctly.